src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -122,19 +129,30 @@ Future<Integer> loadData(TenantAttributes attributes, String tenantId,
         tl.add(p);
       }
       tl.withKey(SAMPLE_KEY).withLead(SAMPLE_LEAD);
-      tl.add("instances", "instance-storage/instances");
       tl.withIdContent();
-      tl.add("holdingsrecords", "holdings-storage/holdings");
-      tl.add("items", "item-storage/items");
-      tl.add("instance-relationships", "instance-storage/instance-relationships");
-      tl.add("bound-with/instances", "instance-storage/instances");
-      tl.add("bound-with/holdingsrecords", "holdings-storage/holdings");
-      tl.add("bound-with/items", "item-storage/items");
-      tl.add("bound-with/bound-with-parts", "inventory-storage/bound-with-parts");
+      tl.add("instances", INSTANCES);
+      tl.add("holdingsrecords", HOLDINGS);
+      tl.add("items", ITEMS);
+      tl.add("instance-relationships", INSTANCE_RELATIONSHIPS);
+      tl.add("bound-with/instances", INSTANCES);
+      tl.add("bound-with/holdingsrecords", HOLDINGS);
+      tl.add("bound-with/items", ITEMS);
+      tl.add("bound-with/bound-with-parts", BOUND_WITH_PARTS);
       tl.withFilter(service -> servicePointUserFilter(service, servicePoints))
         .withPostOnly()
         .withAcceptStatus(422)
-        .add("users", "service-points-users");
+        .add("users", SERVICE_POINTS_USERS);
+      future = future.compose(n -> tl.perform(attributes, headers, vertxContext, n));
+    }
+
+    if (isNew(attributes, "25.1.0")) {
```

Comment:
What happens if this code is executed twice and database content is really 25.1.0 or later? The isNew will return true if module is being enabled after being disabled. Yet the database content with sample data already holds these entries if it was 25.1.0 or later at some point.